frontend/src/app/auth/auth-form/auth-form.component.html

```diff
@@ -0,0 +1,29 @@
+<div class="container">
+    <div class="container__wrapper">
+            <h1>Budgetify</h1>
+            <form [formGroup]="loginForm" (ngSubmit)="onSubmit()" class="auth-form">
+                <mat-form-field class="auth-form__row" appearance="outline" hideRequiredMarker>
+                    <mat-label>Email</mat-label>
+                    <input matInput type="email" formControlName="email">
+                </mat-form-field>
+                <span *ngIf="loginForm.controls['email'].errors?.['required'] && loginForm.controls['email'].touched" class="auth-form__error-message">
```

Comment:
i think this check loginForm.controls['email'].touched is not needed
you can remove in all places

Reply:
Without it error message is displayed from the beginning when login page is opened, which is not a desired behavior.